src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -178,7 +178,9 @@ void GcsServer::DoStart(const GcsInitData &gcs_init_data) {
   // be run. Otherwise the node failure detector will mistake
   // some living nodes as dead as the timer inside node failure
   // detector is already run.
-  gcs_heartbeat_manager_->Start();
+  if (gcs_heartbeat_manager_) {
```

Comment:
can you add a check at least one of `gcs_heartbeat_manager_ `or `gcs_healthcheck_manager_ ` must be nullptr?